content/_index.md

```diff
@@ -1,15 +1,18 @@
 +++
-title = "d3adb5"
 date  = 2022-03-08
```

Comment:
Date should be updated to be consistent with last change.

Reply:
Dates updated in a378f02.